### DISCUSSION TITLE
Question about "Reuse downloaded media"

### DISCUSSION THREAD

**Discussion by CanePlayz** [Tyrrrz/DiscordChatExporter#598] [Q&A] [answered]
I have downloaded a whole server with 20 channels in one folder (there are the HTML files and the folders with the downloaded media) and I want to update my back-ups. With "Reuse downloaded media" on, do I just have to delete the HTML files then and can I keep the folders with the downloaded files because it will download the new files into the already existing folders?

**Comment by Tyrrrz** [comment, marked as answer]
Yes. Also make sure the file names of the exports are the same.

**Reply by CanePlayz** [reply]
Ok thx

**Reply by 96-LB** [reply]
I believe the HTML files will be replaced if you use the same filename so deleting them manually shouldn't be necessary.
